docs/domains/Compliance/services/ValidationService/openapi.yml

```
@@ -413,6 +413,71 @@ paths:
                   $ref: '#/components/schemas/Result'
       security:
       - bearer-key: []
+  /api/validation/result/{facilityId}/{reportId}/category:
+    get:
+      tags:
+      - result-controller
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)